import 'package:dart_custom_exceptions/dart_custom_exceptions.dart';
import 'package:test/test.dart';

void main() {
  test('calculate', () {
    expect(calculate(), 42);
  });
}
